src/Debug.js

```diff
@@ -63,24 +63,29 @@ function indentChildren(childrenStrs, indentLength) {
     : '';
 }
 
-export function debugNode(node, indentLength = 2) {
+export function debugNode(node, indentLength = 2, options = {}) {
   if (typeof node === 'string' || typeof node === 'number') return escape(node);
   if (!node) return '';
 
-  const childrenStrs = compact(childrenOfNode(node).map(n => debugNode(n, indentLength)));
+  const childrenStrs = compact(childrenOfNode(node).map(n => debugNode(n, indentLength, options)));
   const type = typeName(node);
-  const props = propsString(node);
-  const beforeProps = props ? ' ' : '';
-  const nodeClose = childrenStrs.length ? `</${type}>` : '/>';
+
+  let props = '';
+  let beforeProps = '';
+  if (!options.ignoreProps) {
+    props = propsString(node);
+    beforeProps = props ? ' ' : '';
```

Comment:
I think i'd prefer to change line 72 to `const props = options.ignoreProps ? '' : propsString(node);` and then none of the consts need to change to lets.